python/lsst/ts/mtdome/mock_llc/mock_motion/azimuth_motion.py

```diff
@@ -68,15 +68,27 @@
     (LlcMotionState.MOVING, LlcMotionState.CRAWLING): [],
     (LlcMotionState.CRAWLING, LlcMotionState.MOVING): [],
     (LlcMotionState.CRAWLING, LlcMotionState.STOPPED): [],
+    (LlcMotionState.MOVING, LlcMotionState.MOVING): [],
+    (LlcMotionState.MOVING, LlcMotionState.CRAWLING): [],
     (LlcMotionState.MOVING, LlcMotionState.STOPPED): [],
+    (LlcMotionState.CRAWLING, LlcMotionState.CRAWLING): [],
+    (LlcMotionState.CRAWLING, LlcMotionState.MOVING): [],
     (LlcMotionState.MOVING, LlcMotionState.PARKED): STATE_SEQUENCE_STOP_MOTORS,
     (LlcMotionState.CRAWLING, LlcMotionState.PARKED): STATE_SEQUENCE_STOP_MOTORS,
     (LlcMotionState.STOPPED, LlcMotionState.PARKED): STATE_SEQUENCE_STOP_MOTORS,
     (LlcMotionState.MOVING, LlcMotionState.STATIONARY): STATE_SEQUENCE_STOP_MOTORS,
     (LlcMotionState.CRAWLING, LlcMotionState.STATIONARY): STATE_SEQUENCE_STOP_MOTORS,
     (LlcMotionState.STOPPED, LlcMotionState.STATIONARY): STATE_SEQUENCE_STOP_MOTORS,
+    (LlcMotionState.STOPPED, LlcMotionState.STOPPED): [],
+    (LlcMotionState.STATIONARY, LlcMotionState.STATIONARY): [],
```

Comment:
Please consider using tuples instead of lists for these state transition lists, because they are immutable. In other words use `(...)` instead of `[...]`.

The one place this is a nuisance is tuples of a single value require a trailing comma, which hampers maintainability. But I don't think you have any of those.

Reply:
I removed the like-to-like entries and expanded an existing if statement to take those into account. Thanks for the great suggestion.

Reply:
Using tuples instead of lists introduced a lot of mypy errors so I will consider doing that in a future ticket.

Reply:
You probably know this but the type annotation typing.Sequence will probably fix that.

If you switched to typing.Tuple I would expect that to fail; it's a weird annotation for things like named tuples where the # of type of each element is known in advance. I model I try to avoid whenever possible.